docs/references/chrome-extension/overview.mdx

```diff
@@ -27,24 +29,26 @@ The authentication options for a Chrome Extension will differ depending on how t
 | Web3 | | | ✅ |
 
 > [!WARNING]
-> Our Chrome Extension SDK currently does not fully support Sync Host on side panels. The side panel would need to be closed and then reopened to update the authentication status
+> Our Chrome Extension SDK currently does not fully support Sync Host on side panels. Currently, if a user authenticates in your web app, they need to close and reopen the side panel to update their auth status.
 
-## Chrome Extension CRX ID
+## Configure a consistent CRX ID
 
-A Chrome Extension can be identified by its unique Id, in a similar way to how a website can be identified by its domain. You will need to explicitly configure your Chrome Extension's CRX ID or it will change often. If the key changes, it can cause the Chrome Extension to fail. See the [Configure a consistent key](/docs/references/chrome-extension/configure-consistent-crx-id) guide for more information.
+A Chrome Extension can be identified by its unique CRX ID, similar to how a website can be identified by its domain. The CRX ID rotates by default, which can cause errors with the Clerk integration. [Learn how to configure a consistent CRX ID](/docs/references/chrome-extension/configure-consistent-crx-id) so that your extension will have a stable, unchanging key.
 
-## Sync Host
+## Sync your Chrome Extension with your web app
```

Comment:
`## Sync auth status between your Chrome Extension and web app`

The title should probably mention something about auth or auth state.

Reply:
can do!

Reply:
updated! [b089102](https://github.com/clerk/clerk-docs/pull/1004/commits/b089102b7f14df60439d79fd7c6313e1a735050a)